mypy/types.py

```diff
@@ -1288,6 +1328,9 @@ def visit_unbound_type(self, t: UnboundType) -> Type:
     def visit_type_list(self, t: TypeList) -> Type:
```

Comment:
... or try out what happens if `TypeTranslator` isn't `SyntheticTypeVisitor` in another diff.